### PR TITLE
6 view list of comments for a review

### PR DESCRIPTION
**Pull request by kontsio1**

### CODE REVIEW

File: src/apis.js [hunk 1]
```diff
@@ -12,4 +12,11 @@ export const getReview = (review_id)=>{
     return gamesApi.get(`/reviews/${review_id}`).then((res)=>{
         return res.data.review
     })
+}
+
+export const getComments = (review_id)=>{
+    return gamesApi.get(`/reviews/${review_id}/comments`).then((res)=>{
+        console.log(res.data.comments)
```
Comment (lewispricey) [inline src/apis.js:19]: Please remove this console.log

File: src/components/DetailedReviewCard.jsx [hunk 2]
```diff
@@ -28,8 +28,8 @@ export const DetailedReviewCard = ()=> {
             <img src={selectedReview.review_img_url} alt={`The game ${selectedReview.title}`} width="200" height="200"></img>
             <p> {selectedReview.review_body}</p>
             <p> Votes: {selectedReview.votes}</p><button>Vote</button>
-            <p> Cretated at {selectedReview.created_at}</p>
-            <button>View Comments</button>
+            <p> Created at {selectedReview.created_at}</p>
+            <a href={`/reviews/${selectedReview.review_id}/comments`}><button>View Comments</button></a>
```
Comment (lewispricey) [inline src/components/DetailedReviewCard.jsx:32]: While this is working, is there a better way for us to move to another route in react?

When I tested this, the URL wasn't updating in the URL bar.

File: src/components/CommentList.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+           {
+            listOfComments.map(({comment_id, body, review_id, author, votes, created_at})=>{
+                return (
+                    <CommentCard key={comment_id} review_id={review_id} body={body} author={author} created_at={created_at} votes={votes}/>
+                )
+            })
+           }
+           </ul>
+           <button>Add new comment</button>
+           <a href={`/reviews/${review_id}`}><button>Go back to review</button></a>
```
Comment (lewispricey) [inline src/components/CommentList.jsx:38]: Is there another way we can change routes in React?

File: src/components/CommentList.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    if (isLoading) {
+        return (
+            <p> Loading please wait...</p>
+        )
+    }
+    return (
+        <section>
+           <h2> Review Comments </h2>
+           <ul>
```
Comment (lewispricey) [inline src/components/CommentList.jsx:28]: Really like your choice of using an UL here, makes things super clear in the HTML what you're rendering 👍

File: src/components/CommentList.jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+           {
+            listOfComments.map(({comment_id, body, review_id, author, votes, created_at})=>{
```
Comment (lewispricey) [inline src/components/CommentList.jsx:30]: Great use of destructuring in your map to keep your code as readable as possible!

**Review by lewispricey — CHANGES_REQUESTED** [review]
Great work so far Konstantinos!

I've left a few comments on the code, please have a look at those <a> tags and the console.log.

What how you've done the comments deffinitely works. I would say that it would be nice to see the comments on the individual review page to make it clear what review they relate to (like youtube comments, linked in below the content). Have a think about how this could be implemented.

Please raise another PR once you've made the changes 😃

File: src/apis.js (hunk 1 above)
Comment (RichardNeat) [inline src/apis.js:19]: Please remove any stray console logs

**Review by RichardNeat — APPROVED** [review]
Nice work! Looks great. Just a stray console log to remove please. You're also getting a warning in your code that you're missing an element from the dependency array review_id. Have a look at why that might be happening and consider adding. Once those have been looked at go ahead and merge :)

File: src/apis.js (hunk 1 above)
Comment (kontsio1) [reply src/apis.js:19]: perfect thanks

**Pull request merged**